native/src/layout.rs

```diff
@@ -64,7 +64,7 @@ impl<'a> Layout<'a> {
     ///
     /// [`Layout`]: struct.Layout.html
     /// [`Node`]: struct.Node.html
-    pub fn children(&'a self) -> impl Iterator<Item = Layout<'a>> {
+    pub fn children(&self) -> impl Iterator<Item = Layout<'a>> + '_ {
```

Comment:
It's probably better to simply take `self` by value here and drop the anonymous iterator lifetime, given that `Layout` implements `Copy`.